applications/knative/1.18.1/extra-images.txt

```diff
@@ -15,6 +15,7 @@ gcr.io/knative-releases/knative.dev/eventing/cmd/mtchannel_broker:v1.18.1
 gcr.io/knative-releases/knative.dev/eventing/cmd/mtping:v1.18.1
 gcr.io/knative-releases/knative.dev/eventing/cmd/webhook:v1.18.1
 gcr.io/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate:v1.18.0
+gcr.io/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate:v1.18.1
```

Comment:
https://github.com/mesosphere/kommander-applications/pull/3875/files#diff-da4114d49f0fd96f0a63946a59a9abe5cbffb33b2bc9705022a4a4bd3b8cfe5aR53 cmd/cleanup is on 1.18.1